rasa/core/channels/socketio.py

```diff
@@ -203,10 +204,12 @@ async def connect(
 
                 if jwt_payload:
                     logger.debug(f"User {sid} connected to socketIO endpoint.")
+                    return True
                 else:
                     return False
             else:
                 logger.debug(f"User {sid} connected to socketIO endpoint.")
+                return True
```

Comment:
Should the return type in L197 also change from `Optional[bool]` to `bool`?

Reply:
done 👍🏻 Good catch!